cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/ProvisionerStateTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class ProvisionerStateTest {
```

Comment:
Can we add tests to check (1) valid state update (i.e. going from in progress to completed is acceptable, but not the other way), (2) bad input (e.g. what is the expected state with a null summary?)